### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by MrExhilarator**

### CODE REVIEW

File: src/appendResults.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+//displayResults iterates through the 15 search results and 
+//appends the results based on the page the user is currently in
+var displayResults = async (searchResults, page) => {
```
Comment (findayush) [inline src/appendResults.js]: Any specific purpose of making this function async, as we are not awaiting for the callback/promise to be returned

File: ytSimulator.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        <div class="search-results" id="searchResults"></div>
+        <div class="button-container" id="buttonContainer"></div>
+    </div>  
+    <script src="./src/index.js"></script>
+    <script src="./src/paginate.js"></script>
+    <script src="./src/retrieveData.js"></script>
+    <script src="./src/appendResults.js"></script>
+</body>
+
+</html>
```
Comment (findayush) [inline ytSimulator.html:23]: Can you please host this webpage using github pages/any hosting server platform !

File: src/appendResults.js (hunk 1 above)
Comment (MrExhilarator) [reply src/appendResults.js]: My bad, it was a mistake :)

File: ytSimulator.html (hunk 2 above)
Comment (MrExhilarator) [reply ytSimulator.html:23]: I've hosted it, Ayush. You can find the link to the page in readme file

**Review by findayush — APPROVED** [review]

**Pull request merged**